build.fsx

```diff
@@ -141,15 +141,16 @@ Target.create "NuGet" (fun _ ->
           ("PublishRepositoryUrl", "true")
           ("EmbedUntrackedSources", "true")
           ("IncludeSymbols", "true")
-          ("SymbolPackageFormat", "snupkg") ]
```

Comment:
Why is this commented out?

Reply:
So I can't work out why this fails the package build when there are two packages.

Without it we get `FSharp.Data.Core.symbols.nupkg` and `FSharp.Data.symbols.nupkg` correctly built.  

Do you know if it matters either way?

Reply:
From https://docs.microsoft.com/en-us/nuget/create-packages/symbol-packages-snupkg

> The legacy format .symbols.nupkg is still supported but only for compatibility reasons like native packages (see [Legacy Symbol Packages](https://docs.microsoft.com/en-us/nuget/create-packages/symbol-packages)). NuGet.org's symbol server only accepts the new symbol package format - .snupkg.

Reply:
Can you just embed symbols instead of the separate package? That's much less hassle for consumers. Separate packages make Sourcelink hard to use.

Reply:
@baronfel Thanks, I've done this now

Reply:
Thank you!